.github/workflows/oadp-compatibility-check.yml

```diff
@@ -0,0 +1,42 @@
+name: OADP version compatibility check
+
+on:
+  push:
+    branches:
+      - master
+      # add release branches
```

Comment:
We can't do that prior to merge? e.g. PR ?

Reply:
did not understand

Reply:
yes, that is intentional

example: we add restore CRD here, we also need to add in OADP

if this is run during PR, we need simultaneously PRs in oadp and here

this way, we do thing here, and then, only after PR here merges, we have to create PR in oadp

Reply:
Shouldn't this be like:
 - we know there is problem with sync, so we override merge?
 
 Like this we may miss post merge failure and then we need to revert back?

Reply:
If is during PR, we need to open simultaneously PR in oadp, and merge it prior to the one here, meaing oadp will have new NAC stuff prior to NAC. I think is wrong

I preffer to be like this, after merging here, you need to update OADP. more clear path and makes more sense, in my opinion

similar discussion https://github.com/migtools/oadp-non-admin/pull/16#discussion_r1556352449